compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -2174,13 +2184,23 @@ impl<'tcx> FmtPrinter<'_, 'tcx> {
 
         define_scoped_cx!(self);
 
+        let possible_names = vec![Symbol::intern("'t"), Symbol::intern("'s"), Symbol::intern("'r")];
```

Comment:
maybe go from `'a` to `'z` before starting with `'t<n>` :thinking: so

```suggestion
 let possible_names: Vec<_> = ('a'..'z').map(|s| Symbol::intern(&format!("'{s}"))).collect();
````